### PR TITLE
fixes discover feed test

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap [hunk 1]
```diff
@@ -437,14 +437,104 @@ exports[`The Discover tab component Should retrieve the Content Channel Feeds 1`
       </View>
     </View>
     <RCTScrollView
+      ListEmptyComponent={
+        <Err
+          error={
+            [Error: Network error: No more mocked responses for the query: query getFeatureFeed($featureFeedId: ID!) {
```
Comment (vinnyjth) [inline apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap:443]: Looks like this query is still erroring, but there's more data here than before.

**Comment by redreceipt** [comment]
ok so now we're back to where we started

**Comment by redreceipt** [comment]
any idea why we're getting a loading state here?

**Comment by redreceipt** [comment]
@vinnyjth if you're cool with worrying about the load state later, the only reason I touched was to attempt to fix the strange jest error in the bump 2.0 PR.

**Comment by vinnyjth** [comment]
We can fix it later. Fixing that issue will requiring writing some pretty intense mocks.

**Comment by redreceipt** [comment]
can I get that approval

**Comment by vinnyjth** [comment]
I wish when I expressed positive emotions about a PR Github would ahead and approve it for me automatically.

**Review by vinnyjth — APPROVED** [review]

**Comment by redreceipt** [comment]
neuralink will make that happen soon enough

**Comment by richarddubay** [comment]
> I wish when I expressed positive emotions about a PR Github would ahead and approve it for me automatically.

I feel like that's something Mergify should do! If I say "It's good" in a comment it just merges it. 😂

**Pull request merged**